packages/analytics-js-plugins/src/errorReporting/utils.ts

```diff
@@ -144,7 +144,15 @@ const getBugsnagErrorEvent = (
 
 const isRudderSDKError = (event: any) => {
   const errorOrigin = event.stacktrace?.[0]?.file;
-
+  const errorMessage = event.message;
+  if (
+    errorMessage &&
+    typeof errorMessage === 'string' &&
+    (errorMessage.includes('unhandledrejection handler received a non-error') ||
+      errorMessage.includes('unhandledException handler received a non-error'))
+  ) {
+    return false;
+  }
```

Comment:
We'll come back to this change later. So, exclude it from this PR so we can review it again.

Reply:
Addressed